docs/website/mtouch-errors.md

```diff
@@ -1531,6 +1531,55 @@ In some cases, it's a "local network" issue and it can be addressed by adding th
 
 <h3><a name="MT7044"/>MT7044: Main Info.plist path does not specify a UIDeviceFamily.</h3>
 
+<h3><a name="MT7045"/>MT7045: Unrecognized Type: *.</h3>
+
```

Comment:
I know it's after the original commit but, minimally, the `7xxx` docs should state which task(s) are issuing the errors. That would help (provide some context) when only the error code is provided (e.g. in a bug report, without any context) when googling into the document.